pkg/sdk/go/sdk.go

```diff
@@ -116,8 +116,8 @@ type Channel struct {
 
 // SDK contains Mainflux API.
 type SDK interface {
-	// CreateUser registers mainflux user.
-	CreateUser(user User) (string, error)
+	// CreateUser registers mainflux user. It is only allowed to perform by admins.
```

Comment:
This is not really true, If policies are set to accept `*`, then it can be performed by anybody (self-registration).